stylesheets/components/cards.css

```diff
@@ -0,0 +1,137 @@
+/* .cards-container{
+    display: grid;
+    grid-template-columns: repeat(auto-fill, minmax(18.75em, 1fr));
+    align-items: center;
+    justify-items: center;
+    gap: 1.25rem;
+} */
```

Comment:
If this is not needed you can remove it instead of commenting out.

Reply:
Otherwise all looks good. 👍

Reply:
Thank you for pointing it out, I've removed it in the latest commit